src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return function getSum(x) {
```

Comment:
[checklist](https://github.com/mate-academy/js_sum-with-closure/blob/master/checklist.md) 2nd item

Reply:
done:)

Reply:
Checklist #2: `[CODE STYLE] - You don't need to name functions if you return them right away.`
Fix it.

Reply:
ops, i fixed this but incorrect save task